src/components/metadata/index.js

```diff
@@ -106,8 +107,10 @@ const MetaData = ({
                   value={metadata.filepath}
                 />
                 <MetaDataRow
-                  label={`Parameters (${metadata.parameters?.length || '-'}):`}
+                  label={'Parameters:'}
```

Comment:
```suggestion
                  label="Parameters:"
```
No need for this syntax with a raw string